app/components/Configuration.react.js

```diff
@@ -21,10 +22,19 @@ class Configuration extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            isMenuOpen: false
+            isMenuOpen: false,
+            username: cookie.load('db-connector-user')
         }
         this.toggle = this.toggle.bind(this);
         this.close = this.close.bind(this);
+        this.logOut = this.logOut.bind(this);
+
+        if(window.require) {
+            window.require('electron').ipcRenderer.once('username',
```

Comment:
is `if (window.require)` a check to see if we're in electron vs web browser? if so, could we move it into a re-usable `utils.js` file? I believe there is already something there called like `isElectron`

Reply:
`isElectron` is a node-module: https://www.npmjs.com/package/is-electron
but it is not working for us as expected (not sure why). So instead I am checking according to the context required ..

Reply:
got it. then can we remove `isElectron` and redefine it in `utils.js` like:
```
const isElectron = () => Boolean(window.require)
```